nav2_map_server/CMakeLists.txt

```diff
@@ -160,5 +162,5 @@ ament_export_libraries(
   ${library_name}
   ${map_io_library_name}
 )
-ament_export_dependencies(${map_io_dependencies} ${map_server_dependencies})
+ament_export_dependencies(${map_io_dependencies} ${map_server_dependencies} yaml-cpp)
```

Comment:
Nice. Why didn't this bite us before?

Reply:
The vendor structure changed recently :(